tardis/conftest.py

```diff
@@ -1,5 +1,23 @@
+import yaml
+import tempfile
+import tardis
+from tardis.atomic import AtomData
+from tardis.io.config_reader import Configuration
 from astropy.tests.pytest_plugins import *
 
+# For specifying error while exception handling
+from socket import gaierror
+
+try:
+    import dokuwiki
+    dokuwiki_available = True
```

Comment:
same goes here - the things that raise the error need to be in the try, but nothing else.